src/common/components/ssr/__tests__/VisibleAfterComponentMounted.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+// Use react-test-renderer to test state before useEffect takes place:
+// https://stackoverflow.com/questions/61459112/how-to-test-component-before-useeffect-with-testing-library-react
+import { create } from 'react-test-renderer';
+
+import { VisibleAfterComponentMounted } from '../VisibleAfterComponentMounted';
+
+it('is visible after mounting', async () => {
+  const Component = (
+    <VisibleAfterComponentMounted>Hello World</VisibleAfterComponentMounted>
+  );
+  const elem = create(Component);
+  expect(elem.root.findByType('span').props.style).toEqual({
+    visibility: 'hidden',
+  });
+  elem.update(Component);
```

Comment:
came across this: https://reactjs.org/docs/test-renderer.html#testrendereract

Any idea why it is recommened to wrap these with act? Their first example doesn't have it

Reply:
and thinking if `elem` coul use better naming? React docs use testRenderer and testInstance === testRendered.root